sources/packages/backend/apps/api/src/app.aest.module.ts

```diff
@@ -166,6 +167,7 @@ import {
     OverawardControllerService,
     RestrictionSharedService,
     MSFAANumberSharedService,
+    ApplicationOfferingChangeRequestService,
```

Comment:
do we need to import it into aest module?

Reply:
You're right. we don't need them in AEST module.

Reply:
Without adding it here, I get an error stating that Nest cannot resolve dependency related to `ApplicationOfferingChangeRequestService`. Hence, keeping it here as is. Please correct me if I am wrong.

Reply:
It was not throwing an error, When I first reviewed it. Now since you have added the service to `AssessmentControllerService`,  removing it will throw an error